api/libproject/devstart.rb

```diff
@@ -695,22 +598,36 @@ def run
   :fn => lambda { |*args| run_drop_cdr_db(*args) }
 })
 
-Common.register_command({
-  :invocation => "run-cloud-cdr-migrations",
-  :description => "Runs database migrations for cdr schema on the Cloud SQL database for the specified project.",
-  :fn => lambda { |*args| run_cloud_cdr_migrations(*args) }
-})
-
 Common.register_command({
   :invocation => "run-cloud-cdr-data-migrations",
   :description => "Runs data migrations in the cdr schema on the Cloud SQL database for the specified project.",
   :fn => lambda { |*args| run_cloud_cdr_data_migrations(*args) }
 })
 
+def connect_to_cloud_db(cmd_name, *args)
+  unless Workbench::in_docker?
+    exec *(%W{docker-compose run --rm scripts ./project.rb #{cmd_name}} + args)
+  end
```

Comment:
This is the biggest improvement in this PR. Circle runs things within a docker container. Developers run things and have docker installed. This allows the script to check which mode it is running in and then proceed the same way in either case.